### ISSUE TITLE
Inquiries: multi cursor and rectangle mode

### ISSUE THREAD

**Issue by jazzlw** [whitphx/vscode-emacs-mcx#1537]
Hi,
Love awesome emacs!

Two questions: 
1. Are there keyboard shortcuts that interact with the vscode multicursor? Can I get a multicursor in other ways than just meta-clicking?
2. I can't seem to get the rectangle commands to work.   `C-x SPC` turns it on, but none of the three key commands seem to do anything. Am I just doing something dumb?

Thanks!!

**Comment by whitphx** [comment]
> 1. 

No, multi-cursor is VSCode's feature, but not an Emacs feature to emulate, so this extension does not set its keybindings.

> 2.

Not sure, but it should work as below.
![CleanShot 2023-02-17 at 00 44 51](https://user-images.githubusercontent.com/3135397/219416416-daf84080-0ae3-4f98-9aed-7d0e230257c1.gif)

**Comment by katana-unlimited** [comment]
emacs-mcx 利用させていただいています。素晴らしいです

わたしの環境でも矩形コマンドが機能していません
C-x r k (kill-rectangle),  C-x r M-w(copy-rectangle-as-kill), C-x r d(delete-rectangle), C-x r y(yank-rectangle)など。
@whitphx さんの説明にあるC-wで切り取った矩形文字列をC-yで貼り付けると正しくペーストできない（矩形が維持されない）ので
C-x r y(yank-rectangle)が使えない状況に困っています
ALT-x でコマンドラインから上記のコマンドを実行しても反応がありませんでした

上記コマンドが動いている方はいらっしゃいますか?

**Comment by whitphx** [comment]
I didn't understand the issue correctly at the moment because I used the three stroke commands starting with `C-x r ` without `C-x SPC`,
and now know that such commands should work in the rectangle mark mode as well.
It will be fixed in the next release.
